### PR TITLE
Content and navigation for event pages

### PR DESCRIPTION
**Pull request by TheNickWilson**

### CODE REVIEW

File: test/navigation/NavigatorSpec.scala [hunk 1]
```diff
@@ -121,16 +121,109 @@ class NavigatorSpec extends SpecBase with ScalaCheckPropertyChecks with Generato
...
+        "to 'check your answers' when the user answers no" in {
+
+          forAll(arbitrary[UserAnswers]) {
+            answers =>
+
+            val updatedAnswers = answers.set(IncidentOnRoutePage, false).success.value
+
+            navigator.nextPage(IncidentOnRoutePage, NormalMode, updatedAnswers)
+              .mustBe(routes.CheckYourAnswersController.onPageLoad(answers.id))
+          }
```
Comment (ashwinirudrappa) [inline test/navigation/NavigatorSpec.scala]: @TheNickWilson Can we add one more test to cover the None condition for IncidentOnRoutePage.

Comment (TheNickWilson) [reply test/navigation/NavigatorSpec.scala]: :+1:
